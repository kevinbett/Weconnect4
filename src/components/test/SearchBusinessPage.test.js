import SearchBiz from '../SearchBusinessPage'
import { shallow } from "enzyme"

describe('Search business page',() => { 
    it ('Renders search business page', () => { 
        expect(shallow(<SearchBiz/>)).toMatchSnapshot()
    });
} )