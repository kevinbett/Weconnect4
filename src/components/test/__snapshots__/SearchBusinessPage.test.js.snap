// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search business page Renders search business page 1`] = `
<div>
  <Navigation />
  <Search />
</div>
`;
